jobs/JGLOBAL_FORECAST

```diff
@@ -20,15 +26,6 @@ for config in $configs; do
     status=$?
     [[ $status -ne 0 ]] && exit $status
 done
-# Source additional configs
```

Comment:
We need waves in the config for the forecast? Why was this removed?

Reply:
It was moved to config.fcst